addons/resource.language.en_gb/resources/strings.po

```diff
@@ -3338,7 +3338,231 @@ msgctxt "#802"
 msgid "%s of %s available"
 msgstr ""
 
-#empty strings from id 803 to 849
+#. Label of spinner control for choosing the timer type (one time, repeating, ...)
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+msgctxt "#803"
+msgid "Timer type:"
+msgstr ""
+
+#. Label of 'Guide data title search' edit field
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+msgctxt "#804"
+msgid "Search Guide for:"
+msgstr ""
+
+#. Help of 'Guide data title search' edit field
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+msgctxt "#805"
+msgid "Enter the search string to use to lookup matching TV shows in the Guide"
+msgstr ""
+
+#. Label of "Guide data fulltext search" edit field
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+msgctxt "#806"
+msgid "Fulltext search (or Title search only):"
+msgstr ""
+
+#. special day selection value for a timer schedule
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#807"
+msgid "Any day"
+msgstr ""
+
+#. special day selection value for a repeating timer
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+#: xbmc/pvr/timers/PVRTimerInfoTag.cpp
+msgctxt "#808"
+msgid "Every day"
+msgstr ""
+
+#. special channel selection value for a repeating timer
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+msgctxt "#809"
+msgid "Any channel"
+msgstr ""
+
+#. special time selection value for a repeating timer
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+msgctxt "#810"
+msgid "Any time:"
+msgstr ""
+
+#. Label of "Day" edit field
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+msgctxt "#811"
+msgid "Day:"
+msgstr ""
+
+#. Label of new episodes radio button (if user does not want to record the same episode multiple times)
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+msgctxt "#812"
+msgid "Record only new episodes:"
+msgstr ""
+
+#. Label of pre-record spinner
+#: xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp
+msgctxt "#813"
+msgid "Pre-record (minutes):"
```

Comment:
Not true. The current timer dialog does. That's why I did it the same way with the reimplemented timers settings dialg. But ofc I'm fine with fixing this inconsistency with the new timer settings dialog. ;-)